code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -137,11 +137,11 @@
 	return
 
 /mob/living/simple_animal/bot/mulebot/screwdriver_act(mob/living/user, obj/item/I)
+	. = ..()
```

Comment:
Might want to check the return value here. If you harm hit the mule with the screwdriver, then it'll return `null` instead of `TRUE`. I don't think you want to do anything if that happens